js/modules/resolution.go

```diff
@@ -95,14 +128,19 @@ func (mr *ModuleResolver) Lock() {
 	mr.locked = true
 }
 
-func (mr *ModuleResolver) resolve(basePWD *url.URL, arg string) (module, error) {
-	if cached, ok := mr.cache[arg]; ok {
-		return cached.mod, cached.err
-	}
+// fix
```

Comment:
What should we fix here? :thinking:

Reply:
Drop this - there are other comments where it used, but I am not doing this now as it requires Sobek work.